plugins/leemons-plugin-grades/frontend/src/pages/private/Evaluations/EvaluationList.js

```diff
@@ -96,6 +99,9 @@ export default function EvaluationList() {
       ...tag,
       scale: tag.scale.number,
     }));
+    if (store.selectedGrade.type === 'numeric') {
```

Comment:
Shouldn't be sorted always? Letter grades also have the number property

Reply:
Done!